tests/metrics/README.md

```diff
@@ -4,25 +4,19 @@
 Some tests require you to enable the debug mode of cc-oci-runtime. To enable it, please take a look at:
 https://github.com/01org/cc-oci-runtime#18running-under-docker
 
-To run the metrics, just run:
+
+### Run all metrics tests
+To run the metrics, please use Makefile rule `metrics-tests` on the top level of this repository:
 
 ```bash
-$ cd tests/metrics
-$ ./run_docker_metrics
+$ sudo make metrics-tests
```

Comment:
This is what works for me: 
```
./autogen.sh --with-qemu-path=/usr/bin/qemu-lite-system-x86_64 --with-cc-image=/usr/share/clear-containers/clear-containers.img --with-cc-kernel=/usr/share/clear-containers/vmlinux.container
make 
sudo make install 
sudo -E make metrics-tests
```
But yes, I have configured `go` to use it with `sudo`.

Currently docker and cri-o tests work with the installed binaries, so the `make install` should be a needed step before running the metrics.